include/oneapi/dpl/distributed_ranges

```diff
@@ -0,0 +1,29 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
```

Comment:
I believe this part of the copyright header should not be used within DR, since it does not inherit nor use any LLVM work. It should only apply to the standard parallel algorithms in oneDPL.